server/plugin/api.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
```

Comment:
should make it clear in the comments that this provides an implementation of `api.PluginAPI` dependency.